root/enry.py

```diff
@@ -0,0 +1,68 @@
+import ast
+from collections import defaultdict
+import json
+import os
+import platform
+import subprocess
+
+from tqdm import tqdm
+
+from extract import logger
+from treesitter import queries, save_identifiers
+
+unparseable_languages = set()
+
+
+def extract_languages(repo_path: str):
+    """
+    Traverse repository directory, for each programming language file create a json using enry,
+    Write json to jsonl file
+    :param repo_path: path of extracted repository
+    """
+    path = os.walk(repo_path)
+    file_dict = defaultdict(dict)
+    with open("../used_jsons/" + repo_path.split("/")[-1] + ".json", "w") as json_file:
```

Comment:
let's use [pathlib](https://docs.python.org/3/library/pathlib.html) - so you don't need to modify strings for path manually - and it will work on several platforms by default